packages/lesswrong/server/resolvers/postResolvers.ts

```diff
@@ -77,7 +77,7 @@ augmentFieldsDict(Posts, {
       type: 'Int', 
       resolver: (post, _, context) => {
         if (!post.debate) return 0;
-        return getDialogueResponseCount(post)
+        return getDialogueResponseIds(post).length
```

Comment:
nit: this will probably need to change to use the new field (rather than post.debate)